src/fancyline/tty/vt100.cr

```diff
@@ -2,16 +2,23 @@ class Fancyline
   class Tty
     lib LibC
       struct Winsize
-        ws_row : UInt16    #  rows, in characters
-        ws_col : UInt16    #  columns, in characters
-        ws_xpixel : UInt16 #  horizontal size, pixels
-        ws_ypixel : UInt16 #  vertical size, pixels
+        ws_row    : ::LibC::UShort # rows, in characters
+        ws_col    : ::LibC::UShort # columns, in characters
+        ws_xpixel : ::LibC::UShort # horizontal size, pixels
+        ws_ypixel : ::LibC::UShort # vertical size, pixels
       end
 
+      {% unless flag?(:x86_64) && flag?(:darwin)%}
+        puts "Warning: Tty::Vt100#winsize \
```

Comment:
This generates a run-time `puts`.  You can use the compile-time/macro puts instead to output something at compile-time: `{% puts "Something" %}`

Reply:
Also, you don't have to hard-break at 80 columns. Up to 120 is fine on modern screens.

Reply:
Run time puts yes, same thing I did in the lib declaration...

Reply:
Oh it it the thing i did in the lib declaration...